scripts/qa_crowbarsetup.sh

```diff
@@ -2614,7 +2614,11 @@ function custom_configuration()
         ceilometer)
             local ceilometerservice="ceilometer-cagent"
             if iscloudver 6plus ; then
-                ceilometerservice="ceilometer-polling"
+                ceilometerservice="ceilometer-central"
+                if iscloudver 6 ; then
```

Comment:
I think you really need to look at the `$cloudsource` value; this check will work for develcloud6, which we don't want.